packages/core/helper-plugin/lib/src/components/ReactSelect/ReactSelect.js

```diff
@@ -17,7 +18,13 @@ const ReactSelect = ({ components, styles, error, ariaErrorMessage, ...props })
     <Select
       {...props}
       menuPosition="fixed"
-      components={{ ClearIndicator, DropdownIndicator, IndicatorSeparator, ...components }}
+      components={{
+        ClearIndicator,
+        DropdownIndicator,
+        IndicatorSeparator,
+        LoadingMessage,
```

Comment:
I think what you want to overwrite is `LoadingIndicator`, so that they are not displayed twice ...

Reply:
mmh yes and no I guess, it's a good question, maybe one for @maevalienard:

`LoadingIndicator` is the one in the main input, `LoadingMessage` is the one inside the popover, which one do we want?

<img width="403" alt="image" src="https://user-images.githubusercontent.com/71838159/187380990-aee595c9-9ecf-4e72-ad8e-39a80d00c4a6.png">

Reply:
The loading indicator is meant to be overwritten in this case - I think we mess up a11y of the component, if we keep both or replace the message with a loader.

Reply:
do you mean we should leave LoadingMessage as a simple text and LoadingIndicator with a Loader+text? not sure I understand!

Reply:
Yes - this is how `react-select` defines both props.

Reply:
mh since we are using `Loader` which contains a11y specification I didn't think it was an issue

<img width="559" alt="image" src="https://user-images.githubusercontent.com/71838159/187382606-d1731ecc-62d0-43fa-8c00-9550c945825b.png">

Reply:
Uh good question. I'd say LoadingMessage is enough for the visual perception. However, out of curiosity, in terms of accessibility, do you think we need both? 🤔

Reply:
@ronronscelestes That is why I wanted to split out this PR from relations ;) I think it requires a bit more thinking/ testing.

Reply:
if I'm not mistaken, in terms of accessibility, @gu-stav says it would be better to leave `LoadingMessage` as a simple text and `LoadingIndicator` as a `Loader` icon

Reply:
Do we really need both of them? It seems a bit overkill to me

Reply:
after discussing it with @maevalienard it seems a lot to have 2 ways to display the loading state (LoadingIndicator + LoadingMessage)
- we would remove the loading indicator (which anyway doesn't bring any accessibility feature)
- we would rely on loading message for now (which is not adding much for accessibility so far)
- next iteration would be to better handle Loader accessibility in the DS then integrate it to the ReactSelect

sounds good for you both?